internal/printer/print-to-tsx.go

```diff
@@ -230,9 +230,6 @@ declare const Astro: Readonly<import('astro').AstroGlobal<%s, typeof %s`, propsI
 		p.addSourceMapping(n.Loc[0])
 		if n.FirstChild == nil {
 			p.print("{(void 0)")
-		} else if expressionOnlyHasComment(n) {
-			// we do not print expressions that only contain comment blocks
-			return
```

Comment:
Don't remove comments in the TSX output

Reply:
I'm not sure, I was wondering the same as that would unexpectedly remove TS directives like `@ts-ignore` in this case.
Pinging @natemoo-re, maybe he could provide some background

Reply:
I honestly don't remember the context! Guessing it was just a copy + paste from the JS output which has special handling for this case.